en_US/deploy/cluster/introduction.md

```diff
@@ -34,7 +34,13 @@ To provide our customers with a better cluster salability performance, EMQX 5.0
 
 
 
-To better understand how clustering in EMQX works, you can continue to read the [EMQX clustering](../../design/clustering.md). 
+To better understand how clustering in EMQX works, you can continue to read the [EMQX clustering](../../design/clustering.md).
+
+::: tip Note
+
+The cluster size of the open-source version of EMQX does not exceed 3 nodes. The cluster will retain the basic masterless architecture to meet users' needs for reliability and scalability.
```

Comment:
As far as I know, there's nothing enforcing this 3-node limit.

Reply:
I updated based on what's described on this page: https://emqx.atlassian.net/wiki/spaces/MP/pages/1021673502/EMQX+Open+Source+Limitations. 

Does anyone know the confirmed answer?

Reply:
Maybe change to this:

> While there is no strict upper limit, it is advisable to limit the cluster size to three nodes in the open-source version of EMQX. Using only core type nodes, a smaller cluster size typically ensures greater stability.